### DISCUSSION TITLE
[Question] How to Use Child object Positions as Tensors for Observations and Rewards in Isaac Lab

### DISCUSSION THREAD

**Discussion by Hokite** [isaac-sim/IsaacLab#1507] [Announcements]
Hello everyone,

In Isaac Lab, I have verified that robot USD file information such as body and joint details can be accessed through ArticulationCfg.

I also confirmed that this information can be retrieved in tensor form across multiple environments and utilized in observation and reward calculations.

However, as shown in the current pictures, I have created a sphere mesh as a child object of the robot's body (link).

I want to use the position of this sphere in tensor form for observations and rewards. Are there any classes or methods defined that allow this?

I need your help, please
![image](https://github.com/user-attachments/assets/84f081e8-a58e-4a38-bb81-4f405a61ad82)
![image](https://github.com/user-attachments/assets/53aa06c7-e409-4258-adcb-02b7df4f304f)

**Comment by RandomOakForest** [comment]
Great question. Will move this to our _Discussions_ for follow up. Thanks for posting this.
